Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8
+FROM oracle/graalvm-ce:20.0.0-java8
```

Comment:
Hm… I'm not opposed to it but are we confident in the switch from OpenJDK -> GraalVM?

Reply:
welder has been using graalvm....so I think it should be fine...I talked to Albano before about graalvm, he's fine with it too

Reply:
ok